app/views/articles/_sidebar_additional.html.erb

```diff
@@ -18,7 +18,7 @@
         </div>
       </div>
     <% end %>
-    <% cache("main-article-right-sidebar-discussions-#{params[:timeframe]}", expires_in: (params[:timeframe].blank? ? 120 : 360).seconds) do %>
+    <% cache("main-article-right-sidebar-discussions-#{params[:timeframe]}-#{ApplicationConfig['HEROKU_SLUG_COMMIT']}", expires_in: (params[:timeframe].blank? ? 120 : 360).seconds) do %>
```

Comment:
I have no problems as is, but if we intend to use that in more places in the future, maybe it might be worth a `cache_key` helper?

```ruby
def cache_key(path)
  "#{path}-#{ApplicationConfig['HEROKU_SLUG_COMMIT']}"
```

Reply:
Yeah that makes sense. I'm going to commit this for now so that we can use it for upcoming changes right away, but yeah we should do that.